active_directory/test/test_active_directory.py

```diff
@@ -0,0 +1,35 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import copy
+
+# 3p
+from nose.plugins.attrib import attr
```

Comment:
If we decide to make the changes to `active_directory` before merging, we could stop using `nose` and rely on `py.test` instead. Not sure what `py.test` offers similar to `attr` in nature to skip tests given certain attributes.